### PR TITLE
ceph-volume: Raise when no osd_id and no osd_fsid is provided

### PR DESCRIPTION
**Pull request by david-caro**
Currently you would get a variable undefined error (for the tags
variable).

Fixes: https://tracker.ceph.com/issues/50665

## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by dsavineau** [comment]
jenkins test ceph-volume tox

File: src/ceph-volume/ceph_volume/devices/lvm/activate.py [hunk 1]
```diff
@@ -269,6 +269,10 @@ def activate(self, args, osd_id=None, osd_fsid=None):
             tags = {'ceph.osd_id': osd_id, 'ceph.osd_fsid': osd_fsid}
         elif not osd_id and osd_fsid:
             tags = {'ceph.osd_fsid': osd_fsid}
+        else:
+            raise RuntimeError('If you pass an an osd ID you need to pass '
```
Comment (dsavineau) [inline src/ceph-volume/ceph_volume/devices/lvm/activate.py:273]:
```suggestion
            raise RuntimeError('If you pass an osd ID you need to pass '
```

**Review by dsavineau — CHANGES_REQUESTED** [review]

File: src/ceph-volume/ceph_volume/devices/lvm/activate.py (hunk 1 above)
Comment (david-caro) [reply src/ceph-volume/ceph_volume/devices/lvm/activate.py]: :facepalm:

**Comment by david-caro** [comment]
Amended the commit to have signed-off-by

**Comment by david-caro** [comment]
This was superseded by https://github.com/ceph/ceph/pull/43014

**Pull request closed without merging**